sky/services/media/src/org/domokit/media/SoundPoolImpl.java

```diff
@@ -13,6 +13,7 @@
 import org.chromium.mojo.system.DataPipe;
 import org.chromium.mojo.system.MojoException;
 import org.chromium.mojom.media.SoundPool;
+import org.domokit.sky.shell.ResourceCleaner;
```

Comment:
Rather than introducing a dependency from services to shell, we should move this class into another directory/namespace that makes sense as a dependency of services.  For example, `sky/services/common` or something.